Tests/update.Tests.ps1

```diff
@@ -229,4 +229,29 @@ Describe 'prism update' {
             )
         })
     }
-}
\ No newline at end of file
+
+    It 'handles module that only has prerelease version' {
+        GivenPrismFile @'
+{
+    "PSModules": [
+        { "Name": "Carbon.Permissions", "Version": "1.*-*" }

```

Comment:
How does this test work once there is a non-prerelease version of this module?

Reply:
It doesn't. But it's there for now. I added a comment explaining it. The only way to guarantee is to publish our own module to the gallery that only has a prerelease version and I wasn't prepared for nor had the time to do that.